src/main/java/teammates/storage/api/InstructorsDb.java

```diff
@@ -388,7 +389,9 @@ private Instructor getInstructorEntityForRegistrationKey(String key) {
     }
 
     private List<Instructor> getInstructorEntitiesForCourse(String courseId) {
-        return load().filter("courseId =", courseId).list();
+        List<Instructor> instructorReturnList = load().filter("courseId =", courseId).list();
+        Collections.sort(instructorReturnList, Instructor.compareByName);
+        return instructorReturnList;
```

Comment:
Please don't do sorting in the storage layer. You can sort the attributes in the logic layer where it's used.